feat(doc): use snake case names on operation sub options

gnito-org opened the pull request:
It took me too much frustration and digging to accidentally discover this.

alanpoulain commented:
I think this is too much. Adding a note in `Configuring Operations` would be sufficient.

gnito-org commented:
I disagree. It's easier and more easily understandable to show by example than describe it in verbose text. The work is already done. It just needs merging. :smile:

alanpoulain commented:
You can show by example in `Configuring Operations` too.
> It just needs merging.

Sure, but it needs to be maintained too :wink: 

Anyway, this is my opinion, maybe @dunglas or someone else prefers to merge as is.

gnito-org commented:
From your last comment it sounds if the additional header was the primary concern. I removed the header and moved the text and example into the `Configuring Options` section.
